docs/src/modules/components/branding/Page.js

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Box from '@material-ui/core/Box';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Grid from '@material-ui/core/Grid';
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
+import SearchAppBar from 'docs/src/modules/components/branding/SearchAppBar';
+import Footer from 'docs/src/modules/components/branding/Footer';
+
+const theme = createMuiTheme({
+  palette: {
+    text: {
+      primary: '#001E3C',
+    },
+    primary: {
+      main: '#007FFF',
+    },
+    secondary: {
+      main: '#001E3C',
+    },
+    background: {
+      default: '#FFF',
+    },
+    vividBlue: '#00C8FF',
+    greyF3: '#F3F6F9',
+    greyEAE: '#EAEEF3',
+    grey5A: '#5A6978',
+    azure: '#007FFF',
```

Comment:
I think that we can all the color upfront, they should all be needed later down the road:

```suggestion
    vividBlue: '#00C8FF',
    emeral: '#21CC66',
    sizzlingRed: '#FF505F',
    sunglow: '#FFC846',
    
```